vendors/aws/modules/3gpp/ports/cellular/src/cellular_3gpp_api.c

```diff
@@ -148,6 +196,44 @@ static CellularError_t atcmdUpdateMccMnc( CellularContext_t * pContext,
                                           cellularOperatorInfo_t * pOperatorInfo );
 static CellularError_t atcmdQueryRegStatus( CellularContext_t * pContext,
                                             CellularServiceStatus_t * pServiceStatus );
+static CellularATError_t parseT3412TimerValue( char * pToken,
```

Comment:
We will need to write docs for all these functions - can be done in a separate PR later though.

Reply:
A new PR will be created with documentation on internal function and variable.